pallets/pool-registry/src/benchmarking.rs

```diff
@@ -148,9 +161,10 @@ benchmarks! {
 	update_and_execute {
 		let admin: T::AccountId = create_admin::<T>(0);
 		let n in 1..<T as pallet_pool_system::Config>::MaxTranches::get();
+		let m in 0..<T as pallet_pool_fees::Config>::MaxPoolFeesPerBucket::get();
 		let tranches = build_update_tranches::<T>(n);
 		prepare_asset_registry::<T>();
-		create_pool::<T>(n, admin.clone())?;
+		create_pool::<T>(n, m, admin.clone())?;
```

Comment:
NOTE: The number of pool fees impacts the POV size but not the weight of all `pool-registry` calls except for the pool registration.